acceleration/fast_training_tutorial.ipynb

```diff
@@ -35,21 +35,30 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #169.                        for val_step in range(len(val_loader)):
</pre><p>this is causing an error:</p><p> </p><pre class="ql-syntax" spellcheck="false">
[2022-07-10T01:34:43.235Z] Running ./acceleration/fast_training_tutorial.ipynb
[2022-07-10T01:34:43.235Z] Checking PEP8 compliance...
[2022-07-10T01:34:43.235Z] stdin:488:25: B007 Loop control variable 'val_step' not used within the loop body. If this is intended, start the name with an underscore.
[2022-07-10T01:34:43.235Z]                     for val_step in range(len(val_loader)):
[2022-07-10T01:34:43.235Z]                         ^
[2022-07-10T01:34:43.235Z] Error: Try running with autofixes: --autofix.
[2022-07-10T01:34:43.235Z] 
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
That's bizarre, the runner.sh test passed fine on my end. Changing the line to the following should fix it:
`for _ in range(len(val_loader)):`

Reply:
I will fix this in PR #786. Thanks.

Reply:
I have added the patch as you suggested, it works now, thanks!